alica_engine/src/engine/AlicaContext.cpp

```diff
@@ -0,0 +1,70 @@
+
+#include "engine/AlicaContext.h"
+#include "engine/AlicaEngine.h"
+
+namespace alica
+{
+
+constexpr int ALICA_VERSION_MAJOR = 1;
```

Comment:
hm, personally I prefer the ubuntu style of versioning, but i'm not fixed on it

Reply:
This way we have a definitive way of notifying backward compatibility. Ex: 1.x will work with 1.y.
For 2.x we might introduce breaking changes say on how agents communicate.

Reply:
cool!